apps/language_models/scripts/vicuna.py

```diff
@@ -1456,9 +1455,10 @@ def compile(self, download_vmfb=False):
                 else:
                     compilation_prompt = "".join(["0" for _ in range(17)])
 
-                if Path(f"first_{self.precision}.mlir").exists():
-                    print(f"loading first_{self.precision}.mlir")
-                    with open(Path(f"first_{self.precision}.mlir"), "r") as f:
+                fllama_path = f"first_{self.model_name}_{self.precision}.mlir"
```

Comment:
Could you rename it like first_model_path, cuz the model name might differ and "first" would be more clear.